app/connectors/CustomsDeclareExportsMovementsConnector.scala

```diff
@@ -37,6 +37,8 @@ class CustomsDeclareExportsMovementsConnector @Inject()(appConfig: AppConfig, ht
 
   private val logger = Logger(this.getClass)
 
+  private val FailedDependencyStatus: Int = 424
```

Comment:
I would probably use the `Status.FAILED_DEPENDENCY` instead of the _"magic number **424**"_

Reply:
Good idea